srcpkgs/kid3/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'kid3'
 pkgname=kid3
 version=3.9.5
-revision=2
+revision=3
 build_style=cmake
-configure_args="-DWITH_APPS='CLI;$(vopt_if KDE KDE Qt)'
+configure_args="-DBUILD_WITH_QT6=ON -DWITH_APPS='CLI;$(vopt_if KDE KDE Qt)'
  -DWITH_DOCBOOKDIR=/usr/share/xsl/docbook -DWITH_FLAC=$(vopt_if flac ON OFF)
  -DWITH_ID3LIB=$(vopt_if mp3 ON OFF) -DWITH_MP4V2=$(vopt_if mp4 ON OFF)
  -DWITH_VORBIS=$(vopt_if vorbis ON OFF) -DWITH_TAGLIB=$(vopt_if taglib ON OFF)
  -DWITH_CHROMAPRINT=$(vopt_if chromaprint 'ON -DWITH_FFMPEG=ON' OFF)"
-hostmakedepends="pkg-config python3 libxslt $(vopt_if KDE extra-cmake-modules)"
-makedepends="qt5-declarative-devel qt5-multimedia-devel qt5-tools-devel docbook-xsl
+hostmakedepends="pkg-config python3 libxslt $(vopt_if KDE extra-cmake-modules)
+ qt6-tools qt6-base"
+makedepends="qt6-declarative-devel qt6-multimedia-devel qt6-tools-devel docbook-xsl
```

Comment:
No need for qt6-tools-devel in makedepends (or anywhere).

Droping it also removes the need for that post_configure step

Reply:
done!